packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -406,9 +486,9 @@ class HorizontalOverflow extends Foundation<
     };
 
     /**
-     * Handler for the click event fired after next or previous has been clicked
+     * Returns the items width and total width of the scroll region
      */
-    private handleClick(direction: ButtonDirection): void {
+    private getItemsWidthAndTotalWidth(): ItemsWidthAndTotalWidth {
```

Comment:
This seems hard to understand, can create two methods `getItemWidths` and `getAvailableWidth`

Reply:
sure